Makefile

```diff
@@ -47,3 +47,7 @@ unit-test:
 .PHONY: update-atlas-sdk
 update-atlas-sdk: ## Update the atlas-sdk dependency
 	(cd cfn-resources && ./scripts/update-sdk.sh)
+
+.PHONY: generate-mocks
```

Comment:
@gssbzn added go:generate to the file and also this in the Makefile. I am not sure if we want to further automate this? (e.g. running it on every build?)

Reply:
I have no strong opinion here but from experience I found mocks to get out of sync or even have bugs because people would forget to regenerate an interface so we ended up having a test to fail the CI if there were diffs when running this target
https://github.com/mongodb/mongodb-atlas-cli/blob/cabf9a65a6bc7f6602d96f58689e9f21005e4941/.github/workflows/code-health.yml#L114-L137

Also to keep in mind the generation doesn't delete unused files but those are less distracting that having the generate being on a broken state and only relaizing when you need it